src/pages/About/MissionAndPurpose/MissionAndPurpose.scss

```diff
@@ -0,0 +1,47 @@
+.title{
+    //styleName: Desktop/H1;
+    font-weight: 400; 
+    font-style: normal;       
+    font-size: 42px;
+    font-family: Lora;
+    line-height: 48px;
+    letter-spacing: 0em;
+    text-align: left;
+}
+
+.body{
+    //styleName: Desktop/P2;
+    margin-bottom: 75px;
+    font-weight: 400;
+    font-style: normal;
+    font-size: 16px;
+    font-family: Poppins;
+    line-height: 24px;
+    letter-spacing: 0em;
+    text-align: left;
+}
+
+.section-three__body{
+    margin-bottom: 171px;
+    /* stylelint-disable */
```

Comment:
Mike should have fixed this in his latest merge so please pull latest main and merge. Then remove the comments on stylelint